build.gradle

```diff
@@ -143,6 +143,8 @@ task testIntelDeflater(type: Test){
 }
 
 task testSRA(type: Test){
+    jvmArgs '-Dsamjdk.sra_libraries_download=true'
```

Comment:
download of native libraries only happen when ngs-java fails to find native libraries locally. unfortunately, it depends on user environment and cannot be tested here